family_context/settings.py

```diff
@@ -32,6 +32,10 @@
     "ALLOWED_HOSTS", cast=lambda v: [s.strip() for s in v.split(",")], default=[]
 )
 
+CSRF_TRUSTED_ORIGINS = config(
+    "TRUSTED_ORIGINS", cast=lambda v: [s.strip() for s in v.split(",")], default=[]
+)
```

Comment:
Github actions' auto-test fails here, saying that v in this case is a list. 
Maybe switch to `s.strip() for s in v` if that still works.